colin-api/src/colin_api/models/filing.py

```diff
@@ -26,16 +26,17 @@
 from flask import current_app
 from registry_schemas.utils import get_schema
 
-from colin_api.exceptions import (  # noqa: I001
-    FilingNotFoundException,  # noqa: I001
-    GenericException,  # noqa: I001
-    InvalidFilingTypeException,  # noqa: I001
-    OfficeNotFoundException,  # noqa: I001
-    PartiesNotFoundException,  # noqa: I001
-)  # noqa: I001
+from colin_api.exceptions import GenericException  # noqa: I001
+from colin_api.exceptions import InvalidFilingTypeException  # noqa: I001
+from colin_api.exceptions import OfficeNotFoundException  # noqa: I001
+from colin_api.exceptions import PartiesNotFoundException  # noqa: I001
 from colin_api.models import Business, CorpName, Office, Party, ShareObject
 from colin_api.resources.db import DB
-from colin_api.utils import convert_to_json_date, convert_to_json_datetime, convert_to_snake
+from colin_api.utils import (
+    convert_to_date_pacific_time, convert_to_json_date, convert_to_json_datetime, convert_to_snake,)
+
+
+from colin_api.exceptions import FilingNotFoundException  # noqa: I001; noqa: I001; noqa: I001
```

Comment:
this should fail the linter. why is the sort override in here 3x

Reply:
the sort override was from the orginal code. That was the result after running isort. I removed that override btw.